src/collective/easyform/interfaces/easyform.py

```diff
@@ -98,19 +98,22 @@ class IEasyForm(Schema):
         defaultFactory=default_resetLabel,
         required=False,
     )
+    directives.write_permission(method=config.EDIT_ADVANCED_PERMISSION)
     method = zope.schema.Choice(
         title=_(u"label_method", default=u"Form method"),
         description=_(u"help_method", default=u""),
         default=u"post",
         required=False,
         vocabulary="easyform.FormMethods",
     )
+    directives.write_permission(form_tabbing=config.EDIT_ADVANCED_PERMISSION)
     form_tabbing = zope.schema.Bool(
         title=_(u"label_form_tabbing", default=u"Turn fieldsets to tabs"),
         description=_(u"help_form_tabbing", default=u""),
         default=True,
         required=False,
     )
+    directives.write_permission(default_fieldset_label=config.EDIT_ADVANCED_PERMISSION)
```

Comment:
To keep things simple - in case of fieldsets the label of the default fieldset is "Default". "Ask your manager to change it."